server/ComputeSuperpixelFeatures/TrainSuperpixelFeatures.xml

```diff
@@ -0,0 +1,218 @@
+      <longflag>analysis_tile_size</longflag>
+      <default>2048</default>
+    </double>
+    <double>
+      <name>analysis_mag</name>
+      <label>Analysis Magnification</label>
+      <description>The magnification at which the analysis should be performed.</description>
+      <longflag>analysis_mag</longflag>
+      <default>5</default>
+    </double>
+    <double>
+      <name>min_fgnd_frac</name>
+      <label>Minimum foreground fraction</label>
+      <description>The minimum amount of foreground that must be present in a tile for it to be analyzed</description>
+      <longflag>min_fgnd_frac</longflag>
+      <default>0.5</default>
+    </double>
+    <region>
+      <name>region</name>
+      <label>Region</label>
+      <longflag>region</longflag>
+      <description>left,top,width,height of the region of interest.  All -1 means the whole image is used.</description>
+      <default>-1,-1,-1,-1</default>
+    </region>
+    <integer>
+      <longflag>maxRegionSize</longflag>
+      <label>Maximum region size</label>
+      <description>Maximum width and height allowed when processing an image, in order to prevent accidentally running on too large a region.  Use -1 for no limit</description>
+      <default>5000</default>
+    </integer>
+  </parameters>
+  <parameters advanced="true">
+    <label>Autoencoder Training</label>
+    <description>Autoencoder train parameters</description>
+    <integer>
+      <longflag>patchSize</longflag>
+      <label>Patch size</label>
+      <description>Patch size for superpixel region</description>
+      <default>28</default>
+    </integer>
+    <integer>
+      <longflag>epochs</longflag>
+      <label>Number of epochs</label>
+      <description>Number of epochs for autoencoder training</description>
+      <default>50</default>
+    </integer>
+    <integer>
+      <longflag>batchSize</longflag>
+      <label>Batch size</label>
+      <description>Batch size for autoencoder trainnig</description>
+      <default>16</default>
+    </integer>
+    <integer>
+      <longflag>compactness</longflag>
+      <label>Compactness</label>
+      <description>Compactness of SLIC algorithm</description>
+      <default>30</default>
+    </integer>
+    <integer>
+      <longflag>min_fgnd_superpixel</longflag>
+      <label>Minimum foreground pixels in a superpixel</label>
+      <description>Minimum number of foreground pixels in a superpixel </description>
+      <default>30</default>
+    </integer>
+    <double>
+      <longflag>rg_ratio_superpixel</longflag>
+      <label>Red and green ratio for superpixel</label>
+      <description>Red and green ratio for superpixel</description>
+      <default>1.3</default>
+    </double>
+    <double>
+      <longflag>min_var_superpixel</longflag>
+      <label>Minumum variance of superpixel</label>
+      <description>Minumum variance of superpixel</description>
+      <default>0.002</default>
+    </double>
```

Comment:
Added some args to remove background region for superpixel. Simply get the foreground region using red and green ratio and variance of superpixel region.